estate/views/estate_property_offer_views.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Tree (List) view -->
+    <record id="estate_property_offer_view_tree" model="ir.ui.view">
+        <field name="name">estate_property_offer_tree</field>
+        <field name="model">estate.property.offer</field>
+        <field name="arch" type="xml">
+            <tree string="Estate List View">
+                <field name="price"/>
+                <field name="partner_id"/>
+                <button name="confirm_offer" string="Confirm" type="object" icon="fa-check"/>
+                <button name="cancel_offer" string="Cancel" type="object" icon="fa-times"/>
+                <field name="status"/>
+            </tree>
+        </field>
+    </record>
+
+    <!-- Form view -->
+    <record id="estate_property_offer_view_form" model="ir.ui.view">
+        <field name="name">estate_property_offer_form</field>
+        <field name="model">estate.property.offer</field>
+        <field name="arch" type="xml">
+            <form string="Property Offer Form View">
+                <sheet>
+                    <group class="o_col">
```

Comment:
Maybe I'm wrong but I think that classes with "o" are odoo specific styles and "oe" odoo enterprise. So when I tested bootstrap classes to have a good form view, I tried to use `o_col` instead of `col` to have the good textbox field length (like on the other apps) ...